### ISSUE TITLE
K8s support for nanorc

### ISSUE THREAD

**Issue by plasorak** [alessandrothea/nanorc#77]

**Comment by gcrone** [comment]
Hi,
  I have pushed my merged branch to gcrone/k8s. It has some modifications to make it run with Kind under Podman  rather than Docker so it may not be useable as is but you can see the hacks I have made.

**Comment by gcrone** [comment]
I have pushed a new branch gcrone/k8s-2 which is based on plasorak/k8s and actually works for me.
The podman Vs. docker differences are dealt with by a new --podman option so hopefully it should work for other people unlike my old branch.
The main problem I found with the plasorak branch was the tweak_nwmgr_connection_for_k8s() call was replacing data with data[[child_node.name] so the init would fail.
The other changes are to continue with the host name parsing in cfgmgr but just not resolve the hostname instead of returning early from _load(). Also, I set the host name in the deployment's pod template so that it matches the app name rather than being auto generated by k8s.
